geomstats/backend/numpy_linalg.py

```diff
@@ -0,0 +1,62 @@
+"""Numpy based linear algebra backend."""
```

Comment:
There is a rebase problem, this file shouldn't be there. The organization of the backends has changed, it is in different folders now